Resources/doc/custom-id-generators.rts

```diff
@@ -0,0 +1,64 @@
+Custom ID Generators
```

Comment:
The file has the extension `rts`, where it should be `rst`.

Reply:
good catch!